package.json

```diff
@@ -36,6 +36,7 @@
     "docker:phpunit": "yarn docker:compose exec wordpress phpunit --configuration=/var/www/html/wp-content/plugins/jetpack/phpunit.xml.dist",
     "docker:tail": "yarn docker:compose exec wordpress bash -c \"/var/scripts/tail.sh\"",
     "docker:clean": "yarn docker:compose down --rmi all -v && rm -rf docker/wordpress/* docker/wordpress/.htaccess docker/wordpress-develop/* docker/logs/* docker/data/mysql/*",
+    "docker:env": "node -e \"require('fs').createWriteStream( 'docker/.env', { flags: 'a' } );\"",
```

Comment:
you js devs, can't we just do `touch docker/.env` ?

Reply:
I'd love to but that won't work for Windows. ;-)

Reply:
...everything I can run [hidden](https://github.com/Automattic/jetpack/tree/master/docker/bin) from Windows inside the container I'm already running but this darn thing has to run outside the container. :-)